src/style.css

```diff
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  position: relative;
+  font-family: Avenir, Helvetica, sans-serif;
+  margin-top: 0;
+}
+
+.bar-form {
+  position: absolute;
+  width: 100%;
+}
+
+.big-field {
+  position: relative;
+  height: 70px;
+  width: 100%;
+  margin-top: 20px;
+}
+
+.small-field {
+  position: relative;
+  height: 42px;
+  width: 100%;
+  margin-top: 20px;
+}
+
+.big-image {
+  position: absolute;
+  height: 19px;
+  width: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.small-image {
+  position: absolute;
+  height: 11px;
+  width: 11px;
+  left: 13px;
+  top: 15px;
+  bottom: 16px;
```

Comment:
The 'bottom' property is not necessary for the .small-image class. Consider removing it.